tools/database/README.md

```diff
@@ -62,6 +62,18 @@ Then you can call
 `neard database change-db-kind --new-kind Cold change-hot`.
 Notice that even though in your mind this db is cold, in your config this db hot, so you have to pass `change-hot`.
 
+## Compact database
+
+Run manual compaction on the SST files. After running this the performance of database will increase significantly.
```

Comment:
> After running this the performance will increase significantly

What happens if I run it 10 times in a row? Will all operations become instant? :)

Reply:
Not really, from my testing with it, first run takes around 2h and subsequent run it is about 10min, I think it goes through every block of SST file